src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -89,6 +100,17 @@ function InvoiceDetails({
   exportFormat,
   refreshData,
 }) {
+  const history = useHistory();
+  const stripes = useStripes();
+  const { search } = useLocation();
+  const accordionStatusRef = useRef();
+  const { restrictions, isLoading: isRestrictionsLoading } = useAcqRestrictions(
+    invoice.id, invoice.acqUnitIds,
```

Comment:
Could you destructor invoice to obtain id, acqUnitIds, status as it is referred several times in this module

Reply:
sure